### PR TITLE
added unit tests covering content trust for plugin install command

### PR DESCRIPTION
**Pull request by adshmh**
**- What I did**
Added unit tests to plugin install command, covering several content trust cases

**- How I did it**
Added new file `cli/command/plugin/install_test.go`

**- How to verify it**

**- Description for the changelog**
Added new unit tests to plugin install command to cover some content trust cases

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: cli/command/plugin/install_test.go [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+func TestInstallErrors(t *testing.T) {
+	testCases := []struct {
+		description   string
+		args          []string
+		expectedError string
+		installFunc   func(name string, options types.PluginInstallOptions) (io.ReadCloser, error)
+	}{
+		{
+			description:   "insufficinet number of arguments",
```
Comment (vdemeester) [inline cli/command/plugin/install_test.go]: typo: insufficient

File: cli/command/plugin/install_test.go [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+			args:          []string{"foo"},
+			expectedError: "Error installing plugin",
+			installFunc: func(name string, options types.PluginInstallOptions) (io.ReadCloser, error) {
+				return nil, fmt.Errorf("Error installing plugin")
+			},
+		},
+		{
+			description:   "Installation error due to missing image",
+			args:          []string{"foo"},
+			expectedError: "Use `docker image pull`",
```
Comment (vdemeester) [inline cli/command/plugin/install_test.go]: Does it output that ? :thinking: cc @thaJeztah

File: cli/command/plugin/install_test.go [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+			args:          []string{"foo", "--alias", "UPPERCASE_ALIAS"},
+			expectedError: "invalid",
+		},
+		{
+			description:   "invalid plugin name",
+			args:          []string{"UPPERCASE_REPONAME"},
+			expectedError: "invalid",
+		},
+		{
+			description:   "Installation error",
```
Comment (silvin-lubecki) [inline cli/command/plugin/install_test.go]: nit: other descriptions are in lower case

File: cli/command/plugin/install_test.go (hunk 2 above)
Comment (thaJeztah) [reply cli/command/plugin/install_test.go]: Including the backtics? hm, it shouldn't, I think

File: cli/command/plugin/install_test.go [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+			installFunc: func(name string, options types.PluginInstallOptions) (io.ReadCloser, error) {
+				return ioutil.NopCloser(strings.NewReader("")), nil
+			},
+		},
+		{
+			description:    "Install with disable flag",
+			args:           []string{"--disable", "foo"},
+			expectedOutput: "Installed plugin foo\n",
+			installFunc: func(name string, options types.PluginInstallOptions) (io.ReadCloser, error) {
+				assert.Check(t, is.Equal(true, options.Disabled))
```
Comment (silvin-lubecki) [inline cli/command/plugin/install_test.go]: nit:
```go
assert.Check(t, options.Disabled)
```

File: cli/command/plugin/install_test.go (hunk 2 above)
Comment (n4ss) [reply cli/command/plugin/install_test.go]: agree that it shouldn't include backtics

File: cli/command/plugin/install_test.go (hunk 1 above)
Comment (adshmh) [reply cli/command/plugin/install_test.go]: Thank you for the review. I will update the PR to address this.

File: cli/command/plugin/install_test.go (hunk 3 above)
Comment (adshmh) [reply cli/command/plugin/install_test.go]: Thank you for the review. I will update the PR to fix this.

File: cli/command/plugin/install_test.go (hunk 4 above)
Comment (adshmh) [reply cli/command/plugin/install_test.go]: Thank you, fixed.

**Pull request merged**